search-term/readme.md

```diff
@@ -0,0 +1,16 @@
+#Term Generation
+
+When calling on getTermList, supply the Art and Architecture Thesaurus(AAT) ID
+
+Function takes the id and retrieves the hierarchy listing of terms, including all parent terms and two levels of children terms. After removing all extra formatting and html tags, function compiles the hierarchy list as a csv list.
+
+#Example Use
+
+
+Using the ID for marble (rock)
+
+`getTermList(300011443)`
```

Comment:
This is a little pedantic but I have been trying to change these across the board to be correct.  The python style guide wants underscores instead of camel case for methods.    You use camel case for classes.